admin/model_atlas_search_analyzer.go

```diff
@@ -141,7 +141,7 @@ func (o *AtlasSearchAnalyzer) GetTokenizer() map[string]interface{} {
 // and a boolean to check if the value has been set.
 func (o *AtlasSearchAnalyzer) GetTokenizerOk() (map[string]interface{}, bool) {
 	if o == nil {
-		return map[string]interface{}{}, false
+		return map[string]interface{}, false
```

Comment:
@wtrocki just flagging for awareness

Reply:
Ack! There is some extra work needed on the PR to make it final. Leaving it open for consideration and we can pick that up if needed.